openedx/features/enterprise_support/tests/test_utils.py

```diff
@@ -61,24 +61,32 @@ def test_get_enterprise_learner_portals_uncached(self):
         enterprise_customer_user.enterprise_customer.save()
 
         request = mock.MagicMock(session={}, user=self.user)
-        portals = get_enterprise_learner_portals(request)
-        self.assertEqual(len(portals), 1)
-        self.assertDictEqual(portals[0], {
+        # Indicate the "preferred" customer in the request
+        request.GET = {'enterprise_customer': enterprise_customer_user.enterprise_customer.uuid}
```

Comment:
Is it worth having a test case where we don't get the uuid so we have to use the most recently modified one?